resources/trusted_certificate.rb

```diff
@@ -26,6 +26,21 @@
 property :content, String, required: [:create]
 property :sensitive, [true, false], default: true
 
+property :certificates_dir, String, default: lazy {
+  case node['platform_family']
+  when 'debian'
+    '/usr/local/share/ca-certificates'
+  when 'suse'
+    '/etc/pki/trust/anchors/'
+  else # probably RHEL
+    '/etc/pki/ca-trust/source/anchors'
+  end
+}
```

Comment:
Based on my suggestion below, this can be simplified to this. You should also just move this below the `certificate_name` property above.
```suggestion
property :certificates_dir, String, default: lazy { certificate_default_dir }
```